app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -81,11 +81,25 @@ def update
                                                            "in the content view version")
     param :major, :number, :desc => N_("Override the major version number"), :required => false
     param :minor, :number, :desc => N_("Override the minor version number"), :required => false
+
+    param :repos_units, Array, :desc => N_("Specify the list of units in each repo"), :required => false do
+      param :repo_units, Hash, :desc => N_("a hash containing a repo label and list of units"), :required => true do
+        param :label, String, :desc => N_("repo label"), :required => true
+        param :rpm_filenames, Array, :desc => N_("list of rpm filenames to include in published version"), :required => true do
+          param :rpm_filename, Array, :desc => N_("rpm filename"), :required => true
```

Comment:
I think this may confuse things since `rpm_filename` is not actually a param. And given `rpm_filenames` is an array of strings.

Reply:
I see, I can just say "list of rpm filename strings" for `:rpm_filenames` above and delete this line.